pkg/services/rpcsrv/subscription_test.go

```diff
@@ -261,6 +268,39 @@ func TestFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, "my_pretty_notification", n)
 			},
 		},
+		"notification matching contract hash and parameter": {
+			params: `["notification_from_execution", {"contract":"` + testContractHash + `", "parameters":[{"type":"Any","value":null},{"type":"Hash160","value":"449fe8fbd4523072f5e3a4dfa17a494c119d4c08"}]}]`,
```

Comment:
`449fe8fbd4523072f5e3a4dfa17a494c119d4c08` is a priv0 hash, right? Rubles contract transfers 1000 rubles to priv0 at block 5. Use `goodSender.StringLE()` instead of the raw value instead, otherwise it's painless to maintain this test.

Reply:
oh, my bad, it is `testContractHash`, it calls `Init` and sends all the tokens to itself, a perfect unique notification IMO. started to use const